tests/foreman/virtwho/api/test_esx.py

```diff
@@ -438,3 +438,51 @@ def test_positive_deploy_configure_hypervisor_password_with_special_characters(
         assert not target_sat.api.VirtWhoConfig().search(
             query={'search': f"name={form_data['name']}"}
         )
+
+    @pytest.mark.tier2
+    def test_positive_remove_env_option(self, default_org, form_data, virtwho_config, target_sat):
+        """remove option 'env=' from the virt-who configuration file and without any error
+
+        :id: 981b6828-a7ed-46d9-9c6c-9fb22af4011e
+
+        :expectedresults:
+            the option "env=" should be removed from etc/virt-who.d/virt-who.conf
+            /var/log/messages should not display warning message
+
+        :CaseLevel: Integration
+
+        :customerscenario: true
+
+        :CaseImportance: Medium
+
+        :BZ: 1834897
+
+        """
+        command = get_configure_command(virtwho_config.id, default_org.name)
+        deploy_configure_by_command(
+            command, form_data['hypervisor_type'], debug=True, org=default_org.label
+        )
+        virt_who_instance = (
+            target_sat.api.VirtWhoConfig()
+            .search(query={'search': f'name={virtwho_config.name}'})[0]
+            .status
+        )
+        assert virt_who_instance == 'ok'
```

Comment:
Is this the best way to assert the viert_who_instance?

Reply:
> Is this the best way to assert the virt_who_instance?

@ColeHiggins2 virt-who config status have the valuse "No Report Yet" and "OK", before deploy virt-who config ,its status is "No Report Yet", after deploy, it will change to "OK". so should assert its status.